##### Video example from Coursera Course


### Confidence Intervals

##### Best Estimate ± Margin of Error   

##### Best Estimate = Unbiased Point Estimate           $$\hat{p}$$
##### Margin of Error = "a few" Estimated standard Errors
##### "a few" = multiplier from appropriate distribution based on desired confidence level and sample design $$1.96$$

### 95% Concidence Level <-> 0.05 Significance

$$\hat{p} \pm 1.96 \centerdot \sqrt{\frac{\hat{p} (1 - \hat{p})}{n}}$$


### Motts Poll Background
What proportion of parents report they use a car seat for all travel with their toddlr?

Population - Parents with a toddler
Parameter of Interest - A Proportion

Construct a 95% Confidence Interval for the population proportion of parents reporting they use a car seat for all travel with their toddler

In [4]:
import numpy as np

In [26]:
# n is the sample size
# y is the number of parents that responded yes to the question

n = 659
y = 540

In [27]:
# p is the best estimate

p = y/s

In [30]:
# Margin of error
# 1.96 based on 95% confidence interval.
# 

pluss = p + 1.96 * np.sqrt(p *(1 - p)/n)
minus = p - 1.96 * np.sqrt(p *(1 - p)/n)

print(f'({minus}, {pluss})')

(0.7900537499137914, 0.8487929875672404)


In [31]:
# The difference between the + and minus intervals as compared to the best estimate should be the same

print(pluss - p)
print(p - minus)

0.02936961882672451
0.02936961882672451


With 95% confidence the population of parents whith a toddler who use a car seat for all travel is estimated to be between 79.01% and 84.88%